docs/CONTRIBUTING.md

```diff
@@ -136,8 +136,8 @@ talk to:
 
 | Command                                      | Backend                                                           | Use case                                                |
 | -------------------------------------------- | ----------------------------------------------------------------- | ------------------------------------------------------- |
-| `pnpm start:live`                            | https://crates.io                                                 | Testing UI changes with the full live site's data       |
-| `pnpm start:staging`                         | https://staging-crates-io.herokuapp.com                           | Testing UI changes with a smaller set of realistic data |
+| `pnpm start:live`                            | <https://crates.io>                                                 | Testing UI changes with the full live site's data       |
+| `pnpm start:staging`                         | <https://staging-crates-io.herokuapp.com>                           | Testing UI changes with a smaller set of realistic data |
 | `pnpm start`                                 | Static fixture test data in `mirage/fixtures`                     | Setting up particular situations, see note              |
 | `pnpm start:local`                           | Backend server running locally                                    | See the Working on the backend section for setup        |
 | `PROXY_BACKEND=https://crates.io pnpm start` | Whatever is specified in the `PROXY_BACKEND` environment variable | If your use case is not covered here                    |
```

Comment:
Should the indentation here then be adjusted? The last column of these rows is shifted compared to the other rows:
![Column shifted in diff](https://github.com/rust-lang/crates.io/assets/11685886/3eb9fbae-60ea-4233-8a9c-c6946f96ada8)

Reply:
Good catch! Updated.